meta/schemas/device-type.yaml

```diff
@@ -0,0 +1,19 @@
+$id: https://openwrt.org/schema/device-type.yaml
+$schema: http://json-schema.org/draft-07/schema#
+description: Type of device
+title: device-type
+type: string
+enum:
+  - Camera
+  - Emulator
+  - Modem
+  - NAS
+  - Range Extender
+  - Router
+  - Single Board Computer
+  - Travel Router - Battery powered
+  - Travel Router
+  - WiFi AP
```

Comment:
@aparcar @adschm 
Shouldn't we use "Wi-Fi", with the dash inside, the official technology name?

Reply:
@pepe2k Why not just WLAN AP? I don't like using the trademark.

Reply:
Or just "Access Point"?

Reply:
@CodeFetch @adschm we can have just "Access Point" but then what about the "WiFi Router" in the line below? Maybe just "Wireless" instead of WiFi/Wi-Fi/WLAN?

Reply:
@pepe2k I don't have a specific opinion here, this was just meant as a suggestion without a personal preference.
Personally, I'm not a fan of the "device-type" at all, as it's not really well-defined (as this discussion shows); I'd remove it entirely.

So, I'd leave the decision to those who actually "use" this parameter  ...

Reply:
Router (without WLAN), Access Point (Router with WLAN), Ceiling AP, Outdoor AP (linearly polarized antennas), Outdoor CPE (cross-polarized antennas)... Anything else out there? 

Another "important" information for the user is if the router is "plenum-rated" and maybe if it has a metal housing and if it has a rack mounting option.

Reply:
@adschm maybe you're right here, type of device can be in most cases easily guessed based on other, more specific information.

@CodeFetch I believe most people would say that an Access Point is a device with just single (sometimes two) Ethernet, which you connect to existing network for providing wireless access to it. I would never call a Wi-Fi router just Access Point. I'm pretty sure, if we want to be specific, we would call it... a wired router with Access Point feature :)